test/unit/index.js

```diff
@@ -183,10 +194,35 @@ describe('minimal-cipher', function() {
         result.should.deep.equal(obj);
       });
 
-      it('should decrypt a stream', async function() {
-        const data = getRandomUint8();
-        const chunks = await encryptStream({data});
-        chunks.length.should.be.gte(0);
+      it('should decrypt a stream with chunkSize 1 byte', async function() {
+        const data = getRandomUint8({size: 100});
+        const chunks = await encryptStream({data, chunkSize: 1});
+        chunks.length.should.eql(100);
+        for(const chunk of chunks) {
+          chunk.jwe.should.be.a.JWE;
+        }
+        const result = await decryptStream({chunks});
+        result.length.should.be.gte(0);
```

Comment:
Should this length be exactly the size of `data` on L198?

Reply:
because they are both uint8 arrays yes, so I did make this change, but the result could be a binary string or a some other typed array in the future or it could be gzipped etc. https://github.com/digitalbazaar/minimal-cipher/pull/29/commits/5583046fb336464f46cb5ce7b5dd14de660d52ad